Changes

```diff
@@ -114,6 +140,10 @@ API
 - GraphComponent : Added protected `parentChanged()` virtual method (#3080).
 - ImagePlug : Added convenience methods for evaluating global image properties (#3073).
 - GraphComponentPath : Added property for accessing the GraphComponent (#3106).
+- LightFilter : Added LightFilter class used as base for renderer spcific implementations (#3020).
+- TaskMutex : Added TaskMutex as part of the foundation of a revamped caching scheme (#3133).
+- LRUCache : Added a TaskParallel policy (#3154).
+- CompoundDataPlug : Replaced MemberPlug with new NameValuePlug (#3161).
```

Comment:
Let's do this the other way round :

- NameValuePlug : Introduced new plug type for associating a name with a value (#3161).

Breaking Changes
-----------------------

- CompoundDataPlug : Removed MemberPlug and \<insert methods we removed here\>. Use NameValuePlug instead.